tests/postgres_tests/test_constraints.py

```diff
@@ -309,7 +309,7 @@ def test_invalid_index_type(self):
 
     def test_invalid_expressions(self):
         msg = "The expressions must be a list of 2-tuples."
-        for expressions in (["foo"], [("foo")], [("foo_1", "foo_2", "foo_3")]):
+        for expressions in (["foo"], ["foo"], [("foo_1", "foo_2", "foo_3")]):
```

Comment:
If you rebase onto main, this should go away :+1:

Reply:
Thanks!